src/popup/utils/constants.ts

```diff
@@ -8,21 +8,27 @@ import type {
 } from '../../types';
 import { i18n } from '../../store/plugins/languages';
 
+export { default as TIPPING_V1_ACI } from './contracts/tippingV1aci';
+export { default as TIPPING_V2_ACI } from './contracts/tippingV2aci';
+export { default as FUNGIBLE_TOKEN_CONTRACT_ACI } from './contracts/fungibleContractAci';
+export { default as PAIR_CONTRACT_ACI } from './contracts/pairContractAci';
+export { default as ZEIT_TOKEN_CONTRACT_ACI } from './contracts/zeitTokenAci';
+
 export const EXTENSION_HEIGHT = 600;
 export const MAGNITUDE = 18;
 export const SEED_LENGTH = 12;
 export const AETERNITY_CONTRACT_ID = 'aeternity';
 export const AETERNITY_SYMBOL = 'AE';
 
 export const TX_TYPE_MDW = {
-  SpendTx: SCHEMA.TX_TYPE.spend,
-  ContractCallTx: SCHEMA.TX_TYPE.contractCall,
-  ContractCreateTx: SCHEMA.TX_TYPE.contractCreate,
-  NamePreclaimTx: SCHEMA.TX_TYPE.namePreClaim,
-  NameClaimTx: SCHEMA.TX_TYPE.nameClaim,
-  NameBidTx: SCHEMA.TX_TYPE.nameClaim,
-  NameUpdateTx: SCHEMA.TX_TYPE.nameUpdate,
-  PayingForTx: 'payingForTx', // TODO: remove after https://github.com/aeternity/aepp-sdk-js/issues/1583 is resolved
+  spendTx: Tag.SpendTx,
+  contractCallTx: Tag.ContractCall,
+  contractCreateTx: Tag.ContractCreateTx,
+  namePreclaimTx: Tag.NamePreclaimTx,
+  nameClaimTx: Tag.NameClaimTx,
+  nameBidTx: Tag.NameClaimTx,
+  nameUpdateTx: Tag.NameTransferTx,
+  payingForTx: Tag.PayingForTx, // TODO: remove after https://github.com/aeternity/aepp-sdk-js/issues/1583 is resolved
```

Comment:
The mentioned issues should be fixed in v13

Reply:
Removed it.